pages/common/more.md

```diff
@@ -1,12 +1,21 @@
 # more
 
-> Open a file for interactive reading, allowing scrolling and search (in forward direction only).
+> Open a file for interactive reading, allowing scrolling and search.
+> More information: <https://manned.org/more>.
 
 - Open a file:
 
-`more {{source_file}}`
+`more {{path/to/file}}`
 
-- Page down:
+- Open a file at a specific line:
+
+`more {{path/to/file}} +{{line_number}}`
```

Comment:
Oh, hold on here - I've just looked into it more closely, and the following would be more correct:

```suggestion
- Open a file displaying from a specific line:

`more +{{line_number}} {{path/to/file}}`
```

2 things here:

1. the `+{{line_number}}` *must* be before `{{path/to/file}}`
2. It starts displaying the file at the specified line, cutting everything else before it off, so its more correct to say "displaying from a specific line", rather than "open a file at a specific line", which would imply that you could scroll up to previous lines, which isn't the case with `more` as far as I know (but is with `less`).

Reply:
I can confirm that it cuts off everything before that line, however `more {{path/to/file}} +{{line_number}}` works for me too (linux, more 2.36.2)

Reply:
I checked this in Ubuntu focal (Docker) and `more v2.34` needs the `+10` before the file.